src/components/dataPicker2/dataPicker2/header/datePickerHeader.tsx

```diff
@@ -63,12 +63,12 @@ export const DatePickerHeader: FC<DatePickerHeaderProps> = ({
     );
   }, [yearsOptions]);
 
-  const onMonthChange = (changedMonth: DropdownValue) => {
+  const onMonthChange = (changedMonth: DropdownValue | DropdownValue[]) => {
```

Comment:
Can we encapsulate the `DropdownValue | DropdownValue[]` to a single type on a Dropdown level?